docs/snippets/react/my-component-story-use-globaltype.js.mdx

```diff
@@ -1,19 +1,33 @@
 ```js
 // MyComponent.stories.js | MyComponent.stories.jsx | MyComponent.stories.ts | MyComponent.stories.tsx
 
+import React from 'react';
+
+import MyComponent from './MyComponent';
```

Comment:
```suggestion
import { MyComponent } from './MyComponent';
```

Reply:
I'll leave it as it is if you don't mind, as per the comment above.